rsmtool/rsmcompare.py

```diff
@@ -104,6 +104,9 @@ def run_comparison(config_file_or_obj, output_dir):
         else:
             configpath = os.getcwd()
 
+    logger.info('Saving configuration file.')
+    configuration.save(output_dir)
```

Comment:
Can you explain a bit about this? Was it missing before?

Reply:
We were not doing this before: see https://github.com/EducationalTestingService/rsmtool/issues/194